Description: require nonlocal ascjs module in test
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2021-11-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Xascjs/test.js
+++ b/Xascjs/test.js
@@ -1,4 +1,4 @@
-const ascjs = require('./index.js');
+const ascjs = require('ascjs');
 
 const bold = text => `\x1B[1m${text}\x1B[0m`;
 
